src/ContinuumAbsorption/ContinuumAbsorption.jl

```diff
@@ -1,26 +1,18 @@
 module ContinuumAbsorption
+export total_continuum_absorption
 
 using ..Korg: ionization_energies, Species, @species_str, _data_dir # not sure that this is the best idea
-using ..Korg: Interval, closed_interval, contained, contained_slice, λ_to_ν_bound
+using ..Korg: Interval, closed_interval, contained, contained_slice, λ_to_ν_bound, hummer_mihalas_w
 include("../constants.jl") # I'm not thrilled to duplicate this, but I think it's probably alright
 
-# define helper functions
-include("bounds_checking.jl")
+include("bounds_checking.jl") # define helper functions
 include("hydrogenic_bf_ff.jl")
-
 include("absorption_H.jl")
 include("absorption_He.jl")
-
 include("absorption_ff_positive_ion.jl")
-#used for metal bf absorption
```

Comment:
I moved these imports to `absorption_metals_bf.jl`, as there's no overhead to duplicate `using`s and it means that reading the code requires a lot less flipping between files.

I realize that it's not consistent with the imports from Korg above, but I think it's slightly better.